demo/README.md

```diff
@@ -643,3 +644,28 @@ python demo/demo_video_structuralize.py
     --label-map-stdet tools/data/ava/label_map.txt \
     --label-map tools/data/kinetics/label_map_k400.txt
 ```
+
+## Audio Demo
+
+Demo script to predict the audio-based action recognition using a single audio feature. To extract audio features use the `build_audio_features.py` script.
+
+```shell
+python demo/demo_audio.py ${CONFIG_FILE} ${CHECKPOINT_FILE} ${AUDIO_FILE} {LABEL_FILE} [--device ${DEVICE_TYPE}]
+```
+
+Optional arguments:
+
+- `DEVICE_TYPE`: Type of device to run the demo. Allowed values are cuda device like `cuda:0` or `cpu`. If not specified, it will be set to `cuda:0`.
+
+Examples:
+
+Assume that you are located at `$MMACTION2` and have already downloaded the checkpoints to the directory `checkpoints/`,
+or use checkpoint url from `configs/` to directly load corresponding checkpoint, which will be automatically saved in `$HOME/.cache/torch/checkpoints`.
+
+1. Recognize an audio file as input by using a ResnetAudio model with a tsn head on cuda by default.
+
+    ```shell
+    # The demo.mp4 and label_map_k400.txt are both from Kinetics-400
+    python demo/demo_audio.py configs/recognition_audio/audioonly/audioonly_r50_64x1x1_100e_kinetics400_audio_feature.py \
+        checkpoints/best_top1_acc_epoch.pth audio_feature.npy label_map.txt
```

Comment:
Would you please replace the config file and checkpoint path here with:
`configs/recognition_audio/resnet/tsn_r18_64x1x1_100e_kinetics400_audio_feature.py` and
`https://download.openmmlab.com/mmaction/recognition/audio_recognition/tsn_r18_64x1x1_100e_kinetics400_audio_feature/tsn_r18_64x1x1_100e_kinetics400_audio_feature_20201012-bf34df6c.pth`. 
So that users can run the example command w.o. any modification.